### PR TITLE
Shiptestifies Reality Dissassociate Discorder/hallucinations

### PR DESCRIPTION
**Pull request by Yellow-Mushroom**
## About The Pull Request

Largely removes the more gamey/NRP hallucinations like 'cap is a rev' 'I hear flashing in chapel' or stuff like that that just doesn't make sense'

Widely expands the variety in audio hallucinations with new text, new combo's, and just a general expansion in types that can occur so they are less predictable/fit together better/are more accurate to the shiptest experience and not the gamemodey station side

It also slightly edits the probability chances of hallucinations occuring per my personal feel of it, making some less likely ones more common and vice versa. This results in it being less predictable/less spammy

## Why It's Good For The Game

Makes RDD significantly less annoying and much much less likely to break immersion, its really annoying having dead bodies shout at you 'security is cult!!' 10 times a round.

## Changelog
:cl:
tweak: adds a much larger variety of text to hallucinations/Reality Dissassociate Discorder, removes LRP/NRP gamemode related hallucinations. Also tweaks occurrence rates
/:cl:

### CODE REVIEW

**Comment by Yellow-Mushroom** [comment]
image
image
image
image
image
image



some examples of better madlibs that could show up

**Comment by Vexylius** [comment]
Oh heck, that's pretty good. 
I would add some accusation that X has gone mad or something, there's only one "is crazy" one, but otherwise looks good to me
 
Also 

lmao

File: strings/hallucination.json [hunk 1]
```diff
@@ -63,151 +121,196 @@
...
-		"Revenant",
-		"Traitor",
-		"Harm",
-		"Blue APC",
-		"I hear flashing",
-		"Help",
-		"%TARGETNAME%"
+		"crazy",
+		"a traitor!",
+		"trying to overthrow the captain!",
```
Comment (Yellow-Mushroom) [inline strings/hallucination.json:246]:
```suggestion
		"trying to overthrow the captain!",
		"going mad!",
```

**Comment by Yellow-Mushroom** [comment]
got it in ',)

**Comment by MitztheKat** [comment]
this is so good, might make a char with RDD just to see the chaos that happens.

**Comment by TypeVar** [comment]
I know it may not be related, but does this fix mindbreaker toxin not working to reduce RDS' effects?

**Comment by Yellow-Mushroom** [comment]
> I know it may not be related, but does this fix mindbreaker toxin not working to reduce RDS' effects?

this only effects the occurrence rates and contents of hallucinations

File: strings/hallucination.json [hunk 2]
```diff
@@ -39,14 +74,37 @@
 		"EEEeeeeEEEE",
 		"khhhhh",
 		"#@§*&£",
-		"H**p m*",
+		"%*[@#",
+		"77 65 65 64",
+		"Snyert",
+		"Gawwa Gawwa",
+		"FWWWWWEEEEEEE",
+		"They're inside the sun",
```
Comment (MarkSuckerberg) [inline strings/hallucination.json:82]: I wonder who they might be...

**Review by MarkSuckerberg — APPROVED** [review]
seems good seems good

**Comment by MarkSuckerberg** [comment]
LGTM ig

**Pull request merged**